src/index.tsx

```diff
@@ -1,7 +1,11 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
+import "./polyfills";
 import App from "./App";
 
+import "normalize.css";
+
```

Comment:
```suggestion
import "normalize.css";
import React from "react";
import ReactDOM from "react-dom/client";

import App from "./App";
import "./polyfills";
```
normalize.css is actually a library import